include/psmove.h

```diff
@@ -416,9 +423,10 @@ ADDCALL psmove_pair(PSMove *move);
  * \brief Add an entry for a controller paired on another host.
  *
  * \param addr The Bluetooth address of the PS move to add
+ * \param model The hardware model type of the controller
  **/
 ADDAPI enum PSMove_Bool
-ADDCALL psmove_host_pair_custom(const char *addr);
+ADDCALL psmove_host_pair_custom(const char *addr, enum PSMove_Model_Type model);
```

Comment:
Hmm, this changes kind of the public API. Should we add a new method `psmove_host_pair_custom_model();` and have the old function just call that with `Model_ZCM1` as the first parameter? This way, existing library users can still use the old function `psmove_host_pair_custom()` without breakage, while new code can take advantage of the model selection.

Reply:
Fair enough. Fixed.